pkgs/applications/graphics/drawing/default.nix

```diff
@@ -0,0 +1,72 @@
+{ stdenv
+, fetchFromGitHub
+, desktop-file-utils
+, file
+, gobject-introspection
+, pkgconfig
+, appstream-glib
+, meson
+, ninja
+, python3
+, gtk3
+, wrapGAppsHook
+, gettext
+, glib
+, python3Packages
+}:
+
+stdenv.mkDerivation rec {
+  pname = "drawing";
+  version = "0.4.10";
+
+  src = fetchFromGitHub {
+    owner = "maoschanz";
+    repo = "${pname}";
+    rev = "${version}";
+    sha256 = "1i8b7hkv9484c0pr2lag6xkf4mq5b2gwcawppgfkf1l8971zxqik";
+  };
+
+  nativeBuildInputs = [
+    wrapGAppsHook
+    gobject-introspection
+    pkgconfig
+    appstream-glib
+    meson
+    ninja
+    python3
+    gtk3
+    gettext
+    glib
+    python3Packages.wrapPython
+  ];
+
+  buildInputs = [
+    file
+    desktop-file-utils
+    gtk3
+    python3
+    glib
+    gobject-introspection
+  ];
+
+  propagatedBuildInputs = with python3.pkgs; [
```

Comment:
you're missing pycairo

https://github.com/maoschanz/drawing/search?q=cairo&unscoped_q=cairo

Reply:
fixed